### ISSUE TITLE
Add option to opt-in into accessibility to print into CLI

### ISSUE THREAD

**Issue by AlexSkrypnyk** [drevops/behat-steps#756]
Currently, accessibility just prints into console

```
[accessibility] /: 0 violations, 24 passes, 1 incomplete (rules: wcag2a,wcag2aa).                                                                               [accessibility] /: 0 violations, 24 passes, 1 incomplete (rules: wcag2a,wcag2aa).                                                                               [accessibility] /: 0 violations, 24 passes, 1 incomplete (rules: wcag2a,wcag2aa).                                                                               [accessibility] /user/login: 0 violations, 27 passes, 1 incomplete (rules: wcag2a,wcag2aa)                                                                      .                                                                               [accessibility] /user/login: 0 violations, 27 passes, 1 incomplete (rules: wcag2a,wcag2aa)                                                                      .                                                                               [accessibility] /user/password: 0 violations, 27 passes, 1 incomplete (rules: wcag2a,wcag2aa)                                                                   .                                                                               [accessibility] /user/password: 0 violations, 27 passes, 1 incomplete (rules: wcag2a,wcag2aa)      
```

We need a way to make this an opt-in.
